xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml

```diff
@@ -11,19 +11,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
-- name: Add GPG key for magma repository
-  apt_key:
-    url: https://artifactory.magmacore.org:443/artifactory/api/gpg/key/public
-    state: present
-
 - name: Add apt-transport-https
   apt: pkg=apt-transport-https state=present update_cache=yes
 
 - name: Add JFrog key
-  shell: wget -qO - https://facebookconnectivity.jfrog.io/artifactory/api/gpg/key/public | apt-key add -
+  shell: wget -qO - https://linuxfoundation.jfrog.io/artifactory/api/security/keypair/magmaci/public | apt-key add -
 
 - name: Add JFrog repo
   apt_repository:
-    repo: 'deb https://facebookconnectivity.jfrog.io/artifactory/list/{{ repo }}/ {{ distribution }} main'
+    repo: 'deb https://linuxfoundation.jfrog.io/artifactory/{{ repo }} {{ distribution }} main'
```

Comment:
Was the slash intentionally removed here?

Reply:
The slash is not required here.

Reply:
Sorry, I thought that the distribution was part of the link but now I see that it's not